lib/adapters/REST/endpoints/app-bundle.ts

```diff
@@ -41,17 +41,21 @@ export const del: RestEndpoint<'AppBundle', 'delete'> = (
 
 export const create = (
```

Comment:
We are missing `RestEndpoint<'AppBundle', 'create'>` here. That's why we didn't get a type error before.

Reply:
Good Spot :)